server/src/test/java/io/envoyproxy/controlplane/server/serializer/CachedProtoResourcesSerializerTest.java

```diff
@@ -21,13 +20,9 @@ public void shouldKeepCachedProtoWhenSerializingSameV2Message() {
         .setClusterName("service1")
         .build();
 
-    Any serializedEndpoint = serializer.serialize(endpoint, V2);
```

Comment:
Same question as for the other serializer class.

Reply:
removed